### DISCUSSION TITLE
Reticulum and Nostr Bridge

### DISCUSSION THREAD

**Discussion by cameri** [markqvist/Reticulum#92] [General]
Would a bridge between Reticulum and Nostr Relays make sense?

For context, this is the Nostr protocol: https://github.com/nostr-protocol/nostr

I have a couple LoRa and Raspberry Pis lying around and would be interested in building a bridge between the two networks.

Would this be of interest to anyone here?

**Comment by trosel** [comment]
previous comparison discussion here btw https://github.com/markqvist/Reticulum/discussions/59

**Comment by markqvist** [comment]
I probably wouldn't have a usecase for this myself, yet. But still would love to see it working, and if you do decide to give it a shot, I am available to answer any questions you might have!

**Comment by cameri** [comment]
Thanks! I'm going to give this a shot. But first I need to get Reticulum working on my RPis.

**Reply by markqvist** [reply]
Awesome! Let me know what kind of problems you have on the Pis, if any. Usually, you should just be able to do `pip install rns`, and have everything installed and ready in a few seconds.

**Comment by trosel** [comment]
NOSTR privacy https://consentonchain.github.io/blog/posts/nostr-privacy/

**Reply by cameri** [reply]
Thanks for bringing this up! Nostr protocol is aimed at decentralization not privacy. However, it's easy to recognize how users using it would want to preserve their privacy while using such protocol. Nostr is not aiming to be perfect from the beginning, but to evolve to become the protocol that we need by consensus and utilization. The current NIP-04 for direct messages "just works" but doesn't have the properties users expect but get today from apps like Signal.
I'm confident we will solve all those issue in due time.

**Comment by trosel** [comment]
Fiatjaf (nostr creator) asking how reticulum works 

https://nostr.com/nevent1qqsqqqrjuhxmtmas9tvar6wkwkyc63czpzxlnjw8nas5q9deems2xaspzemhxue69uhhyetvv9ujumn0wd68ytnzv9hxgq3q80cvv07tjdrrgpa0j7j7tmnyl2yr6yr7l8j4s3evf6u64th6gkwsfx3c74
